Project4/styles.css

```
@@ -0,0 +1,151 @@
+@import url(https://fonts.googleapis.com/css?family=Montserrat);
+
+$nav-height: 60px;
+h1 h2 h3 {
+  color: #ffc400;
+}
+body {
+  text-align:center;
+  margin:0px;
+  padding:0px;
+  font-family:'Montserrat', 'Oswald', 'Poppins', 'PT Sans',
+  'Titillium Web', 'Nunito', 'PT Sans Narrow',
+  'Rubik', 'Arimo', 'Arvo', 'Cabin', 'Fjalla One',
+  'Abel', 'Kanit', san-serif;
+  color:#001d1f;
+}
+p {
+  text-align:left;
+  margin:0px;
+  padding:20px;
+  font-family:'Montserrat', 'Oswald', 'Poppins', 'PT Sans',
+  'Titillium Web', 'Nunito', 'PT Sans Narrow',
+  'Rubik', 'Arimo', 'Arvo', 'Cabin', 'Fjalla One',
+  'Abel', 'Kanit', san-serif;
+  color:#00ffaa;
+}
+a {
+  color: #061d50
+}
+.header {
+  width:100%;
+  height:100px;
+  font-size:50px;
+  line-height:100px;
+  text-transform:uppercase;
+  font-weight:bold;
+  color: #00ffff
+}
+.button1 {
+  background-color: #008CBA;
+  border: none;
+  color: white;
+  padding: 5px 15px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  font-family:'Montserrat', 'Oswald', 'Poppins', 'PT Sans',
+  'Titillium Web', 'Nunito', 'PT Sans Narrow',
+  'Rubik', 'Arimo', 'Arvo', 'Cabin', 'Fjalla One',
+  'Abel', 'Kanit', san-serif;
+}
+img {
+ width: 100%;
+}
+svg {
+  height: 50px;
+  width: auto;
+}
+.speakers {}
+p {
+  line-height:1.25;
+  font-size: 16px;
+  padding-left: 30%;
+  padding-right: 20%;
+}
+.site-wrapper {
+  color: #59bdff;
+  padding-top: 50px;
+  margin-top: 0px;
+  padding-bottom: 25px;
+  min-width: 375px;
+  margin: 10px auto;
+  background-color: #00184b;}
+  a:hover {
+    color: hotpink;
+  }
+  /*.navbar {
+  width:100%;
+  height:$nav-height; //height of nav div
+  font-size:20px;
+  line-height:$nav-height; //sets line height to same height of div to centre text vertically
+  background: #000000;;
+  color:#80f4f8de;
+  position:relative;
+  margin-bottom:-$nav-height; //negative margin set to div height to stop the page jumping when the sticky fixed class is added
+  z-index:3;
+  padding: 5px 15px;
+  text-align: center;
+}*/
+
+.content {
+  width:640px;
+  height:500vh;
+  font-size:14px;
+  padding-top:100px;
+  margin:0 auto 0;
+}
+
+.sticky {
+  position:fixed;
+  top:0;
+}
+.navbar {
+  overflow: hidden;
+  background: #76efff;
+  position: fixed;
+  top: 0;
+  width: 100%;
+  height: 60px;
+}
+
+.navbar a{
+  display: inline-block;
+  color: #200046;
+  text-align: center;
+  margin: 3px;
+}
+.name {
+  font-family:'Montserrat', 'Oswald', 'Poppins', 'PT Sans',
+  'Titillium Web', 'Nunito', 'PT Sans Narrow',
+  'Rubik', 'Arimo', 'Arvo', 'Cabin', 'Fjalla One',
+  'Abel', 'Kanit', san-serif;
+  font-size: 50px;
+}
+img.speaker {
```

You can just use the classname and skip the element:
```suggestion
.speaker {
```